tests/test_transaction.py

```diff
@@ -7,17 +7,21 @@
     "path",
     [
         SNOWBALL_WHEEL,
-        f"/{SNOWBALL_WHEEL}" f"a/{SNOWBALL_WHEEL}",
+        f"/{SNOWBALL_WHEEL}",
+        f"a/{SNOWBALL_WHEEL}",
         f"/a/{SNOWBALL_WHEEL}",
         f"//a/{SNOWBALL_WHEEL}",
     ],
 )
-@pytest.mark.parametrize("protocol", ["https:", "file:", "emfs:", ""])
+@pytest.mark.parametrize(
+    "protocol",
+    ["http:", "https:", "file:", "emfs:", ""],
```

Comment:
It stands for "Emscripten file system". It's needed because `file:` points to files in the actual file system. Probably could use some comments and better documentation.